app/views/site/_study_description_view.html.erb

```diff
@@ -33,3 +36,12 @@
   </script>
 
 <% end %>
+<script type="text/javascript" nonce="<%= content_security_policy_script_nonce %>">
+  $("#toggle-study-resources").click(function() {
+    $("#study-sidebar-container").toggleClass('closed')
+    $("#toggle-study-resources > span").toggleClass('fa-angle-right fa-angle-left')
+
+  })
+
+  $('.corresponding-author').click(function() { showAuthorPopover(this) })
+</script>
```

Comment:
That's a really good point -- we're in no rush on this since we're at the beginning of the sprint.  So I'll hold off merging this for a bit and see if I have time post-Vite migration to migrate some of this to react